src/libstd/sys/unix/ext/process.rs

```diff
@@ -63,3 +64,49 @@ impl ExitStatusExt for process::ExitStatus {
         }
     }
 }
+
+#[stable(feature = "from_raw_os", since = "1.1.0")]
+impl FromRawFd for process::Stdio {
+    /// Creates a new instance of `Stdio` from the raw underlying file
+    /// descriptor.
+    ///
+    /// When this `Stdio` is used as an I/O handle for a child process the given
+    /// file descriptor will be `dup`d into the destination file descriptor in
+    /// the child process.
+    ///
+    /// Note that this function **does not** take ownership of the file
+    /// descriptor provided and it will **not** be closed when `Stdio` goes out
+    /// of scope. As a result this method is unsafe because due to the lack of
+    /// knowledge about the lifetime of the provided file descriptor, this could
```

Comment:
I feel slightly uneasy about this comment, as it directly contradicts the [documentation for the trait](http://static.rust-lang.org/doc/master/std/os/unix/io/trait.FromRawFd.html).

It would be good to be more crisp about the story here.